guides/common/modules/con_troubleshooting-postgresql.adoc

```diff
@@ -30,7 +30,7 @@ If a database was previously created using PostgreSQL 10, perform an upgrade:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} postgresql-upgrade
+# {client-package-install} postgresql-upgrade
```

Comment:
This is on the server, so I think `project-package-install` is more appropriate:
```suggestion
# {project-package-install} postgresql-upgrade
```